push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/UnifiedPushMessage.java

```diff
@@ -33,6 +33,9 @@
  * <pre>
  *  "message": {
  *   "alert": "HELLO!",
+ *   "title": "Safari Title",
```

Comment:
It's actually not just Safari title - Apple Watch will use that too (and I think iPhone etc as well, with iOS 8.2)